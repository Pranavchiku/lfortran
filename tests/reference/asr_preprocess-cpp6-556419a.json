{
    "basename": "asr_preprocess-cpp6-556419a",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cpp6.f90",
    "infile_hash": "656ffe7987384117872854f94922370f2be857b4aacef14bc6c74347",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-cpp6-556419a.stdout",
    "stdout_hash": "3d8125b4a1dfc0f52c286cf73c622a65a5abc7991741f6cca23624d2",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}