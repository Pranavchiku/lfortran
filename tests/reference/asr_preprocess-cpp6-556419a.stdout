(TranslationUnit (SymbolTable 1 {cpp6: (Program (SymbolTable 2 {z12: (Variable 2 z12 Local () () Default (Real 4 []) Source Public Required .false.)}) cpp6 [] [(= (Var 2 z12) (Cast (IntegerConstant 5 (Integer 4 [])) IntegerToReal (Real 4 []) (RealConstant 5.000000 (Real 4 []))) ()) (Print () [(StringConstant "OK" (Character 1 2 () []))] () ())])}) [])
